Xamarin.Forms.Platform.iOS/Platform.cs

```diff
@@ -355,11 +372,10 @@ internal void LayoutSubviews()
 				return;
 
 			var rootRenderer = GetRenderer(Page);
+			rootRenderer?.SetElementSize(new Size(_renderer.View.Bounds.Width, _renderer.View.Bounds.Height));
 
-			if (rootRenderer == null)
-				return;
-
-			rootRenderer.SetElementSize(new Size(_renderer.View.Bounds.Width, _renderer.View.Bounds.Height));
+			if (_pageOverlayView != null && !ReferenceEquals(_pageOverlayView, _renderer.View.Subviews.LastOrDefault()))
+				throw new InvalidOperationException("Overlay must be the topmost subview of MainPage.");
```

Comment:
This should ensure any future code changes to `LayoutSubviews` do not add a subview atop the overlay.